x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/components/policy_form_layout.test.tsx

```diff
@@ -0,0 +1,353 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { PolicyFormLayout } from './policy_form_layout';
+import '../../../../../../common/mock/match_media.ts';
+import { EndpointDocGenerator } from '../../../../../../../common/endpoint/generate_data';
+import {
+  AppContextTestRender,
+  createAppRootMockRenderer,
+} from '../../../../../../common/mock/endpoint';
+import { getPolicyDetailPath, getEndpointListPath } from '../../../../../common/routing';
+import { policyListApiPathHandlers } from '../../../store/test_mock_utils';
+import { licenseService } from '../../../../../../common/hooks/use_license';
+
+jest.mock('../../../../../../common/hooks/use_license');
+
+describe('Policy Form Layout', () => {
```

Comment:
This entire file is just a refactor, right? (move from prior location?)
There is so much here that can be simplified given the new capabilities we have in mocking APIs and in AppContextTestRender. example: we should no longer need to use `sleep`. 
Ok to leave it as is for now. we can address code simplification in the future.

Reply:
Exactly, just moved from one side to another